scripts/migration/migrate_deleted_wikis.py

```diff
@@ -0,0 +1,43 @@
+import logging
+import sys
+
+from modularodm import Q
+
+from framework.transactions.context import TokuTransaction
+from website.app import init_app
+from website.models import NodeLog
+from scripts import utils as script_utils
+
+logger = logging.getLogger(__name__)
+
+def get_targets():
+    return NodeLog.find(Q('action', 'eq', NodeLog.WIKI_DELETED))
+
+def migrate(targets, dry_run=True):
+    # iterate over targets
+    for log in targets:
+        node = log.node
+        versions = node.wiki_pages_versions
+        current = node.wiki_pages_current
+        updated_versions = {}
+        for wiki in versions:
+            if wiki in current:
+                updated_versions[wiki] = versions[wiki]
+        node.wiki_pages_versions = updated_versions
```

Comment:
This migration could use some logging. At the very least, any nodes whose wiki_pages_versions was changed should be logged. Something like:

```
Modified Node 123ab wiki_pages_versions from {'abc123..': [....} to {....}
```

It would also be nice to include a count of modified nodes.